x-pack/legacy/plugins/siem/public/components/timeline/body/actions/index.tsx

```diff
@@ -137,6 +166,7 @@ export const Actions = React.memo<Props>(
       prevProps.eventId === nextProps.eventId &&
       prevProps.eventIsPinned === nextProps.eventIsPinned &&
       prevProps.loading === nextProps.loading &&
+      prevProps.loadingEventIds === nextProps.loadingEventIds &&
```

Comment:
I think that will use the `isEqual` because if we are using an immutable array, it might not be different.

Reply:
In testing this didn't seem to cause an issue and I was seeing the necessary re-renders. I was trying to keep the `isEqual`'s to a minimum, and noticed that existing code was using `===` on arrays from redux with success. Let's change it though if we'll miss necessary renders.